hkmc2/jvm/src/test/scala/hkmc2/JSBackendDiffMaker.scala

```diff
@@ -116,3 +122,13 @@ abstract class JSBackendDiffMaker extends MLsDiffMaker:
       
       
 
+
+class JSBuilderSanityCheck extends JSBuilder:
```

Comment:
It would be best to make this trait take a runtime parameter called `instrument: Bool`, so that whether the instrumentation is done can be configured at runtime (in the difftests or elsewhere).